src/flick/like/views.py

```diff
@@ -44,3 +52,44 @@ def post(self, request, pk):
             self._create_comment_like_notification(user, comment.owner, comment)
         comment_data = CommentSerializer(comment, context={"request": request}).data
         return success_response(comment_data)
+
+
+class LstLikeView(generics.GenericAPIView):
+    permission_classes = api_settings.CONSUMER_PERMISSIONS
+
+    def _create_lst_like_notification(self, from_user, to_user, lst):
```

Comment:
you don't want a notification if you like the list yourself, are you making this precaution? :o

Reply:
I already have that checked : P